### PR TITLE
Update pick-right-solution.md

### PR DESCRIPTION
**Pull request by avigailof**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by jaredbhatti** [comment]
@avigailof  Please sign the SLA, thanks!

**Comment by avigailof** [comment]
i signed the CLA with a different email and now added this email address to my profile. thanks! please recheck

**Comment by mistyhacks** [comment]
It is not picking up your SLA. :( Please see https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting.

**Comment by avigailof** [comment]
I signed it

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -206,6 +208,7 @@ These solutions are combinations of cloud providers and operating systems not co
 * [Nutanix AHV](https://www.nutanix.com/products/acropolis/virtualization/)
 * [OpenShift Container Platform](https://www.openshift.com/products/container-platform/) (OCP) Kubernetes platform by [Red Hat](https://www.redhat.com)
 * [oVirt](/docs/setup/on-premises-vm/ovirt/)
+* [Platform9 Managed Kubernetes as a Service](https://platform9.com/managed-kubernetes/)works on any infrastructure: on-premises, bare metal, or private/hybrid cloud.
```
Comment (mistyhacks) [inline content/en/docs/setup/pick-right-solution.md]: You need a space before "works"

File: content/en/docs/setup/pick-right-solution.md [hunk 2]
```diff
@@ -218,6 +221,7 @@ These solutions are combinations of cloud providers and operating systems not co
 * [Fedora (Multi Node)](/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)
 * [Kubernetes on Ubuntu](/docs/getting-started-guides/ubuntu/)
 * [OpenShift Container Platform](https://www.openshift.com/products/container-platform/) (OCP) Kubernetes platform by [Red Hat](https://www.redhat.com)
+* [Platform9 Managed Kubernetes as a Service](https://platform9.com/managed-kubernetes/)works on any infrastructure: on-premises, bare metal, or private/hybrid cloud.
```
Comment (mistyhacks) [inline content/en/docs/setup/pick-right-solution.md]: Here too

File: content/en/docs/setup/pick-right-solution.md (hunk 1 above)
Comment (avigailof) [reply content/en/docs/setup/pick-right-solution.md]: fixed, thanks!

File: content/en/docs/setup/pick-right-solution.md (hunk 2 above)
Comment (avigailof) [reply content/en/docs/setup/pick-right-solution.md]: fixed, thanks!

**Pull request merged**